.pre-commit-config.yaml

```diff
@@ -374,3 +374,7 @@ ci:
   skip:
     - markdown-link-check
     - poetry-lock
+  - repo: https://github.com/gitleaks/gitleaks
+    rev: v8.16.3
+    hooks:
+        - id: gitleaks
```

Comment:
This can't just be added to the end, since it doesn't belong in the `ci` section.

Created a separate PR #57 for this.

```suggestion
```